Backend/routes/symptoms.py

```
@@ -19,80 +13,43 @@ def add_symptom():
     if not (week and symptom):
         return jsonify({"error": "Missing week_number or symptom"}), 400
 
-    db.execute('INSERT INTO weekly_symptoms (week_number, symptom, note) VALUES (?, ?, ?)', (week, symptom, note))
-    db.commit()
+    try:
+        db_service.save_symptom_entry(week, symptom, note)
+        return jsonify({"status": "success", "message": "Symptom added"}), 201
+    except Exception as e:
```

Ruff (0.14.14)

Do not catch blind exception: `Exception`

(BLE001)